cartography/intel/gcp/__init__.py

```diff
@@ -27,4 +115,12 @@ def start_gcp_ingestion(session, config):
             e
         )
         return
-    crm.sync_gcp_organizations(session, credentials, config.update_tag, common_job_parameters)
+    resources = _initialize_resources(credentials)
+
+    # If we don't have perms to pull Orgs or Folders from GCP, we will skip safely
```

Comment:
Is this comment accurate? I can't see any code explicitly handling this case.

Reply:
Added code handling this